### PR TITLE
Issue#95

### PR DESCRIPTION
**Pull request by FaxriddinMaxmadiyorov**
@XaoGao Hello, the issue is complete, could you please check?

### CODE REVIEW

**Comment by FaxriddinMaxmadiyorov** [comment]
@XaoGao kindly reminding ...

**Comment by o-200** [comment]
@FaxriddinMaxmadiyorov, hi, can you write some tests for confirmation of logic?

`show resume if resume is open or user is admin or moderator`

**Comment by XaoGao** [comment]
@FaxriddinMaxmadiyorov pls write test, authorize user before action, in policy user is will be exist

**Comment by FaxriddinMaxmadiyorov** [comment]
@XaoGao Hello, could you please check?

File: spec/policies/resume_policy_spec.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+require "rails_helper"
```
Comment (XaoGao) [inline spec/policies/resume_policy_spec.rb:1]: @FaxriddinMaxmadiyorov pls use dsl https://actionpolicy.evilmartians.io/#/testing?id=rspec-dsl

**Review by XaoGao — COMMENTED** [review]
@FaxriddinMaxmadiyorov fix spec

**Comment by FaxriddinMaxmadiyorov** [comment]
@XaoGao Hello, I rewrote in RSpec DSL

**Pull request merged**